composer.json

```diff
@@ -56,26 +60,25 @@
             "@t3g:test:php:unit"
         ],
         "t3g:cgl": [
-            "php-cs-fixer fix --config=.php_cs-fixer.dist.php -v --dry-run"
+            "Build/tools/php-cs-fixer/vendor/bin/php-cs-fixer fix --config=.php_cs-fixer.dist.php -v --dry-run"
         ],
         "t3g:cgl:fix": [
-            "php-cs-fixer fix --config=.php_cs-fixer.dist.php"
+            "Build/tools/php-cs-fixer/vendor/bin/php-cs-fixer fix --config=.php_cs-fixer.dist.php"
         ],
         "post-autoload-dump": [
```

Comment:
that seems to be wrong for v12 see 

https://github.com/benjaminkott/bootstrap_package/blob/master/composer.json#L105-L106